src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeService.kt

```diff
@@ -47,6 +57,43 @@ class VilkårperiodeService(
         )
     }
 
+    fun hentVilkårperioderResponse(behandlingId: UUID): VilkårperioderResponse {
+        val grunnlagsdataVilkårsperioder = hentEllerOpprettGrunnlag(behandlingId)
+
+        return VilkårperioderResponse(
+            vilkårperioder = hentVilkårperioderDto(behandlingId),
+            grunnlag = grunnlagsdataVilkårsperioder,
+        )
+    }
+
+    private fun hentEllerOpprettGrunnlag(behandlingId: UUID): VilkårperioderGrunnlag? {
```

Comment:
Kan ev vurdere å opprette dette samtidig som vi oppretter annen grunnlagsdata i `opprettGrunnlagsdataHvisDetIkkeEksisterer`
Vi kommer ha grunnlagsdata for ytelser også, så kan være fint hvis grunnlagsdata opprettes på et sted kanske?

Reply:
Forsåvidt enig, men risikerer vi ikke at grunnlagsdata ikke er opprettet når man henter vilkårsperioder 🤔

Reply:
Eller, vi rendrer kanskje ikke inngangsvilkår siden før hentBehandling er fullført så det går nok fint

Reply:
Tar evt. dette i en senere PR for å få ut denne.